### ISSUE TITLE
Keyboard navigation should be added

### ISSUE THREAD

**Issue by tamertemizer** [team-phoenix/Phoenix#153]
User should be able to navigate the library with arrow keys.

**Comment by athairus** [comment]
Covered under #58.
